client-v2/src/shared/components/snapLink/SnapLink.tsx

```diff
@@ -47,7 +49,25 @@ const SnapLinkURL = styled.p`
   overflow: hidden;
 `;
 
-const ImageWrapper = styled.div``;
+const PageViewDataWrapper = styled.div`
+  width: 45px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  padding-right: 5px;
+  font-size: 12px;
+  height: 35px;
+  padding-bottom: 3px;
+`;
```

Comment:
Are these defined somewhere else as well? Could we pull them out into their own files and import them into both places?

Reply:
They are indeed. Good idea.